agimus_controller_ros/agimus_controller.py

```diff
@@ -86,10 +86,10 @@ def wait_first_sensor_msg(self):
             self.rate.sleep()
         return wait_for_input
 
-    def wait_twice_control_horizon_from_plan(self):
-        for _ in range(self.params.horizon_size * 2):
-            tp = self.hpp_subscriber.get_trajectory_point()
-            # self.buffer.append
+    # def wait_twice_control_horizon_from_plan(self):
```

Comment:
if you comment you remove the code.

Reply:
I commented it because it's making a pre-commit warning but I guess we're gonna use it later, should we still remove it ?

Reply:
In principle leaving undead code is bad practice.
Either you fix it or you remove it, and if needed you use git in order to retrieve later on.